src/assets/scss/mts/modules/content-header/_content-header.scss

```diff
@@ -16,6 +16,7 @@
 
   &__title {
     @include bold-48;
+    word-break: break-all;
```

Comment:
Checking how other libraries handle this, the following seems to be the accepted pattern to break words.

```css
overflow-wrap: break-word;
word-wrap: break-word; // IE
```